packages/paste-website/src/pages/foundations/data-visualization/index.mdx

```diff
@@ -0,0 +1,170 @@
+
+<Callout>
+  <CalloutTitle as="h3">Be mindful about how you apply this palette.</CalloutTitle>
+  <CalloutText>
+    The colors of this palette should be applied in sequence strictly as described below. The sequence is carefully
+    curated to maximize contrast between neighboring colors to help with visual differentiation. You can start anywhere
+    in the sequence, but be sure to follow the remaining order (i.e. You can start with 5, as long as you use the rest
+    of the sequence in order; 6, 7, 8, 9, 10, 1, 2, 3, 4 …)
+  </CalloutText>
+</Callout>
+
+<DataVizColorSwatches />
+
+## Chart anatomy
+
+Each element included in a chart plays an important role in data communication. In this section we’ll provide recommendations on how to style and locate the auxiliary elements such as chart titles, axis titles, legends, etc. More detail will be provided soon!
+
+## Chart guidelines
+
+Choosing the most appropriate chart type is critical in telling the most convincing stories around data. In this section we’ll be providing recommendations on which charts to use and when. More detail will be provided soon!
+
+## Additional accessibility for charts
+
+### Divider lines
+
+Although we’ve ensured that the recommended [data visualization colors](#color-palettes) are 3:1 accessible against the background, it’s not possible to keep them 3:1 accessible against each other at the same time. So for charts that are using the categorical palette, we recommend using a divider line to separate any touching colors which also help to differentiate between categories.
+
+An example showing the use of divider lines is [this pie chart categorizing the race diversity index in the U.S. from the 2010 census data](https://www.census.gov/content/dam/Census/newsroom/blogs/2021/rsblog-meauring-race-ethnic-diversity-2020-census/rsblog-measuring-race-ethnic-diversity-2020-census-figure4.jpg). A white divider line is used to create contrast and help distinguish the difference between the categories.
+
+<Img fluid={props.data.pieChart.childImageSharp.fluid} />
+<PieChartCaption />
+
+### Shapes
+
+Shapes can also be used in combination with color coded data points. The main considerations when designing these are ease of differentiation and consistency of use. Just like semantic colors, certain shapes can have semantic meaning too (e.g. triangles for warning or caution). For this reason, it’s important for teams to align on their use, especially in status alerts.
```

Comment:
```suggestion
Shapes can also be used in combination with color-coded data points. Using shapes in addition to color adds another layer of differentiation so that charts are not relying solely on color to communicate meaning.
```